azureVMCreate.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Quickstart: Use the Azure CLI to create a Linux VM'
+description: In this quickstart, you learn how to use the Azure CLI to create a Linux virtual machine
+author: cynthn
+ms.service: virtual-machines
+ms.collection: linux
+ms.topic: quickstart
+ms.workload: infrastructure
+ms.date: 06/01/2022
+ms.author: cynthn
+ms.custom: mvc, seo-javascript-september2019, seo-javascript-october2019, seo-python-october2019, devx-track-azurecli, mode-api
+---
+
+# Quickstart: Create a Linux virtual machine with the Azure CLI
+
+**Applies to:** :heavy_check_mark: Linux VMs
+
+This quickstart shows you how to use the Azure CLI to deploy a Linux virtual machine (VM) in Azure. The Azure CLI is used to create and manage Azure resources via either the command line or scripts.
+
+In this tutorial, we will be installing the latest Debian image. To show the VM in action, you'll connect to it using SSH and install the NGINX web server.
+
+If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
+
+## Launch Azure Cloud Shell
+
+The Azure Cloud Shell is a free interactive shell that you can use to run the steps in this article. It has common Azure tools preinstalled and configured to use with your account. 
+
+To open the Cloud Shell, just select **Try it** from the upper right corner of a code block. You can also open Cloud Shell in a separate browser tab by going to [https://shell.azure.com/bash](https://shell.azure.com/bash). Select **Copy** to copy the blocks of code, paste it into the Cloud Shell, and select **Enter** to run it.
+
+If you prefer to install and use the CLI locally, this quickstart requires Azure CLI version 2.0.30 or later. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI]( /cli/azure/install-azure-cli).
+
+## Create a resource group
+
+Create a resource group with the [az group create](/cli/azure/group) command. An Azure resource group is a logical container into which Azure resources are deployed and managed. The following example creates a resource group named *myResourceGroup* in the *eastus* location:
+
+```bash
+az group create --name myResourceGroup --location eastus
+```
+
+## Create virtual machine
+
+Create a VM with the [az vm create](/cli/azure/vm) command.
+
+The following example creates a VM named *myVM* and adds a user account named *azureuser*. The `--generate-ssh-keys` parameter is used to automatically generate an SSH key, and put it in the default key location (*~/.ssh*). To use a specific set of keys instead, use the `--ssh-key-values` option.
+
+```bash
+az vm create \
+  --resource-group myResourceGroup \
```

Comment:
We should never have hard coded values.

Reply:
The example here was to show that we do in fact support the pattern of hard coded values today. The thought was not supporting this pattern may turn some doc writers off who have written all of their samples with hard coded values. Probably worth discussing what we want to do here. Perhaps we can mention at the top of the document that this is supported but not recommended?